ostelco-ios-client/ViewControllers/Home/ApplePayViewController.swift

```diff
@@ -24,14 +24,14 @@ class ApplePayViewController: UIViewController {
     var applePayError: ApplePayError?
 
 }
-extension ApplePayViewController: ApplePayDelegate {
+extension ApplePayViewController: ApplePayDelegate, PKPaymentAuthorizationViewControllerDelegate {
```

Comment:
fwiw i'd break this out into separate extensions - that way you can more clearly tell which are the apple methods you're overriding

Reply:
Since I am only using default implementation for

Reply:
extension ApplePayViewController: ApplePayDelegate is empty,